core/domain/exp_domain.py

```diff
@@ -587,6 +601,11 @@ def from_dict(
                 state_domain.WrittenTranslations.from_dict(
                     sdict['written_translations']))
 
+            state.image_assets = (
+                state_domain.ImageAssets.from_dict(
```

Comment:
```
state.image_assets = state_domain.ImageAssets.from_dict(
    sdict['image_assets'])
```

Reply:
done

Reply:
is it done?

Reply:
In general, make sure you have resolved the comment before replying "Done!".